CHANGELOG.md

```diff
@@ -33,21 +33,25 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nidcpower` (NI-DCPower)
     * #### Added
+        * `get_channel_names` - [#1386](https://github.com/ni/nimi-python/issues/1386) 
         * API parity with NI-DCPower 20.7.0 by adding Output Cutoff functionality.
-        * Properties added:
-            * `output_cutoff_current_change_limit_high`
-            * `output_cutoff_current_change_limit_low`
-            * `output_cutoff_current_measure_limit_high`
-            * `output_cutoff_current_measure_limit_low`
-            * `output_cutoff_current_overrange_enabled`
-            * `output_cutoff_enabled`
-            * `output_cutoff_voltage_change_limit_high`
-            * `output_cutoff_voltage_change_limit_low`
-            * `output_cutoff_voltage_output_limit_high`
-            * `output_cutoff_voltage_output_limit_low`
-        * Methods added:
-            * `clear_latched_output_cutoff_state`
-            * `query_latched_output_cutoff_state`
+            * Properties added:
+                * `output_cutoff_current_change_limit_high`
+                * `output_cutoff_current_change_limit_low`
+                * `output_cutoff_current_measure_limit_high`
+                * `output_cutoff_current_measure_limit_low`
+                * `output_cutoff_current_overrange_enabled`
+                * `output_cutoff_enabled`
+                * `output_cutoff_voltage_change_limit_high`
+                * `output_cutoff_voltage_change_limit_low`
+                * `output_cutoff_voltage_output_limit_high`
+                * `output_cutoff_voltage_output_limit_low`
+            * Methods added:
+                * `clear_latched_output_cutoff_state`
+                * `query_latched_output_cutoff_state`
+    * #### Changed
+    * #### Removed
+        * `get_channel_name` and `get_channel_name_from_string` - [#1386](https://github.com/ni/nimi-python/issues/1386)
```

Comment:
This is not shown in the diff. I think you need to delete this line and create an issue that we need to deprecate the two functions.
`nimi-python` has no support for deprecation yet :(

Reply:
I'll remove it from the changelog. But I had already [created a user story](https://ni.visualstudio.com/DevCentral/_workitems/edit/1393704) for the deprecation. Should there be both a user story and an issue, or is the user story sufficient?

Reply:
For `nimi-python` all development is done out in the open, so we use GitHub for the project management.

Reply:
[Done](https://github.com/ni/nimi-python/issues/1586).